### DISCUSSION TITLE
安装后没有任何变化

### DISCUSSION THREAD

**Discussion by Pecuer** [the1812/Bilibili-Evolved#3334] [提问 / Q&A] [answered]
Google Chrome
版本 101.0.4951.67（正式版本） （64 位）
Tampermonkey
4.16
反复安装了好多遍 预览版和正式版都试了
B站旧版新版都不可以

**Comment by Geeyun-JY3** [comment, marked as answer]
装功能了吗？

认真阅读 README 并跟着步骤走了吗？
https://github.com/the1812/Bilibili-Evolved#设置
> 全新安装的脚本实际上没有任何功能, 你可以通过多种方式添加功能

**Reply by Pecuer** [reply]
你好 安装脚本并启用后 在网页左侧中央并没有 功能面板和设置面板的入口

**Reply by Pecuer** [reply]
![2022-05-18_170446](https://user-images.githubusercontent.com/88835949/169002273-787d7ef2-b444-493f-966b-f9afc2eb1e40.jpg)![2022-05-18_170501](https://user-images.githubusercontent.com/88835949/169002282-3cb4fc55-1d0b-4f5e-801a-4913704d41d6.jpg)

**Reply by the1812** [reply]
去开个 issue 吧，需要看看控制台信息，我感觉又是 jsdelivr 连不上了

**Reply by Pecuer** [reply]
谢谢

**Reply by RushQ** [reply]
同样的情况，目前没有找到办法

**Reply by RushQ** [reply]
Microsoft Edge
版本 101.0.1210.53 (正式版本) (64 位)
Tampermonkey
4.16.1
反复安装了好多遍 预览版和正式版都试了
B站旧版新版都不可以

**Reply by the1812** [reply]
见 #3331 
被墙了
